src/sinks/metrics/datadog.ts

```diff
@@ -113,3 +214,10 @@ interface DatadogMetric {
   points: [number, number][]; // [timestamp, value]
   tags: string[];
 }
+
+interface DatadogPoint {
+  metric: string;
+  type: string;
+  points: [number, number[]][]; // [timestamp, [values]]
+  tags: string[];
+}
```

Comment:
Wondering if should just merge `DatadogMetric` and `DatadogPoint` and just have a `points` type which can be `[number, number | number[]][]` or more verbose as `[number, number][] | [number, number[]]`

Reply:
Will update soon